setup.cfg

```diff
@@ -35,19 +35,21 @@ flask =
 	Flask-Compress
 	Flask-Cors
 tornado =
-    tornado
+	tornado
 dev = 
 	black
 	bump2version
 	check-manifest
 	flake8
 	h5grove[fastapi]
 	h5grove[flask]
+	h5grove[tornado]
 	invoke
 	mypy
 	myst-parser
 	# Needed for fastapi tests. Could be removed by setting fastapi[all] as dep in the future.
 	httpx >= 0.23
+	pydantic_settings
```

Comment:
Had to install this manually. Now realising this was removed in https://github.com/silx-kit/h5grove/pull/82, so let me know if I need to revert.

Reply:
dash or underscore doesn't make a difference (underscore gets converted to dash)

`pydantic_settings` should not be needed here since the `[dev]` dependencies includes `h5grove[fastapi]` which includes `pydantic-settings`, so maybe something wrong here.

Reply:
Strange. Oh well, I've reverted.

Reply:
Hmm now it fails `black` formatting in the CI :sweat_smile:

Reply:
Had an older version of `black` locally. Not sure how the CI passed before...